spec/features/homepage_spec.rb

```diff
@@ -0,0 +1,25 @@
+require 'rails_helper'
+
+feature 'Homepage' do
+  # comment this out because there may be changes on how we associate the data. 
+  #
+  # context 'products exist' do
+  #   before do
+  #     @desk = create(:product)
+  #     @amazon = create(:retailer)
+  #     create(:retailer_product, product: @desk, retailer: @amazon)
+  #   end
```

Comment:
@hiattp I'm not sure about this, are we using only Product table? or still using Retailer Product table to keep a relationship between retailers and products.